### ISSUE TITLE
Update Board of Directors

### ISSUE THREAD

**Issue by tehstun** [CodeDoesGood/org-website#32]
The board of directors needs to be updated on the team.html page under the legacy code base.

- [ ] Clone the repro
- [ ] Create a new branch of your naming style, following the current policy  https://github.com/CodeDoesGood/business/wiki/3.-Git-Procedures
- [ ] using your editor, head into the team.html (legacy/website/views/team.html) and update the board of directors page to match the current line up (below)
- [ ] Commit the change and push it to your branch
- [ ]  Make the pull request
- [ ] Get feedback and repeat from step 4, otherwise your all done!

Seek all information you need about each board of director as you go along. 

You will need:
 - Twitter Account
 - Github Account
 - LinkedIn Account
 - Title
 - Full Name
 - Image

Some of these might not be accessible and would have to be left out.
List of current board of directors will be sent to you directly via slack

**Comment by tehstun** [comment]
@apandey09 Hows it going?

**Comment by apandey09** [comment]
@tehstun Going well, I suppose. Hope your week's been fine as well. Just following the instructions you posted for me in Slack. In other words, unless you tell me to go ahead push the changes onto here, I still haven't gone past committing all the changes I made last Sunday.

**Comment by tehstun** [comment]
@apandey09 ideally if you are still working on the branch and this task, then you should be committing the changes and pushing. This allows others to see what is happening while also i'm able to spin up the web server as well to have a look.

**Comment by apandey09** [comment]
@tehstun Right, I understand that. It's just that, as far as I know, your instructions were not to push any changes I committed onto the develop branch. But I'd be willing to do that if you wanted me to, or would you like for me to create a new branch and push my changes onto that one?

**Comment by tehstun** [comment]
you should still push to your previous branch

**Comment by apandey09** [comment]
Alright, but before I do that, there are couple of I'd like to point out. I ran a "git status" on git bash and saw this message. Also, as shown in the image below, there appears to only one branch: the develop branch. The branch I made seems to no longer exist. 
![gitbash2](https://user-images.githubusercontent.com/20345876/38445631-2ce19440-39a9-11e8-9405-b13643684b25.JPG)

**Comment by tehstun** [comment]
okay, so you are behind, which means you need to pull. 

- [ ] git pull
- [ ] create new branch
- [ ] commit changes onto new branch

If you have issues with 1. then let me know, we will have to stash your changes and then come back to them after 3.

**Comment by apandey09** [comment]
For the first step, is it just git pull? Or do I also include the name of the branch from which I would like to pull the changes from, which in this case would be "git pull origin develop"?

**Comment by tehstun** [comment]
just git pull on the developer branch, if you are using the command line instead of a client you can do:
1. git checkout develop
2. git pull

hope that helps :)

**Comment by apandey09** [comment]
Thanks for the info! So after I ran 'git pull', I ran git status to see what new changes were there and this is what I got 
![gitpull](https://user-images.githubusercontent.com/20345876/38524503-a6f1594e-3c03-11e8-882c-df27857de710.JPG)

**Comment by apandey09** [comment]
After seeing that, I thought this would be a good time to move on to the next steps and create a new branch so I can commit those changes onto there. However, when I tried checking out to the new branch I made, I read this message:
![gitcheckoutnewbranch](https://user-images.githubusercontent.com/20345876/38524605-0fdc6804-3c04-11e8-90d6-7957df99c60d.JPG)

**Comment by tehstun** [comment]
you should think about using a git client like git kraken or something simular

**Comment by tehstun** [comment]
Nothing is working because you have file conflicts, you will need to stash the current files and then git pull , you are still not on the most up to date version

**Comment by tehstun** [comment]
You cannot move on to anything until you resolve the conflicts and get the up to date master branch

**Comment by apandey09** [comment]
So I ran git stash and got a message that read "cannot save the current index state".

**Comment by tehstun** [comment]
you are in a merging state, you need to understand what you are doing.

1. you need to complete the merge (stated on every line)
2. You need to know what you are doing: https://medium.freecodecamp.org/useful-tricks-you-might-not-know-about-git-stash-e8a9490f0a1a
3. Please look into using a git client, it will help you out alot when new to git, ([GitKraken](https://www.gitkraken.com/), [SourceTree](https://www.sourcetreeapp.com/), [TortoiseGit](https://tortoisegit.org/))

**Comment by apandey09** [comment]
Hey! Just wanted to update you on my latest progress. I added the files in question that needed to be merged and am no longer in a merging state. As a result, I am now able to checkout to the branch I made. I have not committed the changes yet, or pulled any for that matter, because I have to stash them first. But before I do that, I just want to make sure if I stash the changes on to the branch I made. Or do I make a different branch and stash them on there, or do I stash them all on GitKraken?

Speaking of which, in case I haven't told you this before,  I've had GitKraken for about a few months now. I've been able to open repos on there as well as commit changes onto those repos. I also recently opened the repo for the develop branch of org-website on Gitkraken, but I haven't committed any of the latest changes on to them yet because I need to stash, and then pull, them first.

**Comment by tehstun** [comment]
Hows it going?

It might be a idea to take the files out (which you updated since the last commit) nuke the repro, clone again and copy the files back in.

**Comment by apandey09** [comment]
Fine man! Nice to hear from you. Unless I misunderstood you, I unstaged (or should I have pulled?) all the files I staged recently from the org-website repo on GitKraken.  Now, by "nuke" the repo, do you mean delete?

**Comment by tehstun** [comment]
Yeah if you would delete the repro and reclone it, you should be at a good base to create a new branch and do the css adjustment from there :)
